conf/uno.conf

```diff
@@ -45,7 +45,7 @@ export FLUO_YARN_TARBALL=fluo-yarn-$FLUO_YARN_VERSION-bin.tar.gz
 # If set, 'uno fetch' will build (instead of downloading) an Accumulo tarball
 # from that directory and copy it to the downloads directory.
 
-#export ACCUMULO_REPO=
+export ACCUMULO_REPO=/home/rkturn2/git/accumulo
```

Comment:
I assume checking this config in was unintentional?

Reply:
Yes it was unintentional and I fixed it, did it again, and then fixed it again.  I miss the old way that Uno worked with the example files.  I am always fighting with git with this file... stashing, unstashing it, having to pick files when  I commit instead of doing all...

Reply:
There are ways you can avoid running into that. Some of the things I do include:

1. Using `git status` before committing, with color output, so changes stand out.
1. Using `git diff HEAD` to verify changes before committing.
1. I also keep master branch pristine, and have a local branch with my local edits; when changes are made to master, I merge to my branch. When I wish to make changes, I do `git checkout -b feature-branch-name origin/master`.

Other suggestions to actually ignore changes to the file can be found at: https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

Reply:
Re 1,2,4,and,5  I usually keep a close eye on what I am pushing for the initial PR and when I push to upstream.  For the commits during the PR, I tend to go with whats fastest.

Re 3.  I can give that a try.  Seems like may be more work than the example file, but I will see.